trinity/utils/chains.py

```diff
@@ -135,12 +135,11 @@ def construct_trinity_config_params(
     if args.data_dir is not None:
         yield 'data_dir', args.data_dir
 
-    if args.nodekey_path and args.nodekey:
-        raise ValueError("Cannot provide both nodekey_path and nodekey")
-    elif args.nodekey_path is not None:
-        yield 'nodekey_path', args.nodekey_path
-    elif args.nodekey is not None:
-        yield 'nodekey', decode_hex(args.nodekey)
+    if args.nodekey:
```

Comment:
Can you change this to:

```suggestion
    if args.nodekey is not None:
```

I *think* that technically, there aren't any bad things to expect with the plain falsey check so this is mostly for consistency with the rest of the code.

Reply:
Fixed it, commits squashed, thanks for the feedback